### ISSUE TITLE
I want to make contact us page 

### ISSUE THREAD

**Issue by bhaskarkumar1** [MonalikaPatnaik/UMatter#115]
please assign me this task as there is no contact us page there. At footer there is no contact us page linked. So i will make one contact us page and add all social medias links in the footer section

**Comment by kasturimandlik** [comment]
I would like to create a contact us for the website.
The contact us page will collect details like mail address, name, contact number, message, etc
I would like to enhance the user experience by changing the ui and making it more responsive
I would like to work on this under GSSOC'23. please assign me this issue.
@MonalikaPatnaik

**Comment by MonalikaPatnaik** [comment]
Hey @bhaskarkumar1..is it done? zit's been long otherwise will assign it to someone else!

**Comment by bhaskarkumar1** [comment]
No, I am working on it , because of college exams I got busy. Give me 1 day time I will request a pull request. Currently I am working on it. @MonalikaPatnaik

**Comment by MonalikaPatnaik** [comment]
Thanks for the PR!

**Comment by bhaskarkumar1** [comment]
you are welcome
